articles/cosmos-db/partition-data.md

```diff
@@ -68,106 +68,6 @@ Azure Cosmos DB is designed for predictable performance. When you create a conta
 > 
 > 
 
-<a name="designing-for-partitioning"></a>
```

Comment:
Makes sense about removing the code, but I'm wondering where to put it. Even though you do it slightly differently per data-model, I think keeping all the partition samples in the same article makes sense - otherwise if we start creating a separate doc per task, per data-model we'll have a million articles. So I'd suggest a new topic under How To Guides/Develop and call it Partition in the TOC.

Reply:
@rafats I think we really need all the code in this article. As it stands, a lot of customers don't understand how to do this, and having this in a separate article will make it harder to find ...